### ISSUE TITLE
Crash on startup with 1.2.0

### ISSUE THREAD

**Issue by patricks** [psequel/psequel-issues#48]
I have updated to 1.2.0 and now the app crashes on startup.
It looks like you didn't include the libodbc.2.dylib

Here is the error message:
```
Process:               PSequel [72346]
Path:                  /Applications/PSequel.app/Contents/MacOS/PSequel
Identifier:            com.psequel.PSequel
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           PSequel [72346]
User ID:               501

Date/Time:             2015-07-06 08:41:24.722 +0200
OS Version:            Mac OS X 10.10.4 (14E46)
Report Version:        11
Anonymous UUID:        783CFD29-5356-6A5C-4961-D81212A824A6

Sleep/Wake UUID:       DEFB55CF-6926-4CEA-8350-0A545F84D00E

Time Awake Since Boot: 94000 seconds
Time Since Wake:       320 seconds

Crashed Thread:        0

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/lib/libodbc.2.dylib
  Referenced from: /Applications/PSequel.app/Contents/MacOS/PSequel
  Reason: image not found

Binary Images:
    0x7fff672b7000 -     0x7fff672ed837  dyld (353.2.1) <72A99D0F-0B56-3938-ABC5-67A0F33757C4> /usr/lib/dyld

Model: MacBookPro12,1, BootROM MBP121.0167.B07, 2 processors, Intel Core i7, 3.1 GHz, 16 GB, SMC 2.28f7
Graphics: Intel Iris Graphics 6100, Intel Iris Graphics 6100, Built-In
Memory Module: BANK 0/DIMM0, 8 GB, DDR3, 1867 MHz, 0x02FE, 0x4544464232333241314D412D4A442D460000
Memory Module: BANK 1/DIMM0, 8 GB, DDR3, 1867 MHz, 0x02FE, 0x4544464232333241314D412D4A442D460000
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x133), Broadcom BCM43xx 1.0 (7.15.166.24.3)
Bluetooth: Version 4.3.5f8 15969, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
Serial ATA Device: APPLE SSD SM0256G, 251 GB
USB Device: Internal Memory Card Reader
USB Device: DELL USB Laser Mouse
USB Device: Bluetooth USB Host Controller
Thunderbolt Bus: MacBook Pro, Apple Inc., 27.1
```

**Comment by wernerlaude** [comment]
mee too
Path:                  /Applications/PSequel.app/Contents/MacOS/PSequel
Identifier:            com.psequel.PSequel
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        ??? [1]
Responsible:           PSequel [601]
User ID:               501

Date/Time:             2015-07-06 12:28:59.912 +0200
OS Version:            Mac OS X 10.10.4 (14E46)
Report Version:        11
Anonymous UUID:        5F23619E-3135-CE08-D2D7-91F0CA99AA3F


Time Awake Since Boot: 59 seconds

Crashed Thread:        0

Exception Type:        EXC_BREAKPOINT (SIGTRAP)
Exception Codes:       0x0000000000000002, 0x0000000000000000

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
  Library not loaded: /usr/local/lib/libodbc.2.dylib
  Referenced from: /Applications/PSequel.app/Contents/MacOS/PSequel
  Reason: image not found

Binary Images:
    0x7fff689c1000 -     0x7fff689f7837  dyld (353.2.1) <72A99D0F-0B56-3938-ABC5-67A0F33757C4> /usr/lib/dyld

Model: MacBookPro11,1, BootROM MBP111.0138.B15, 2 processors, Intel Core i5, 2.4 GHz, 8 GB, SMC 2.16f68
Graphics: Intel Iris, Intel Iris, Built-In
Memory Module: BANK 0/DIMM0, 4 GB, DDR3, 1600 MHz, 0x802C, 0x384B54463531323634485A2D314736453120
Memory Module: BANK 1/DIMM0, 4 GB, DDR3, 1600 MHz, 0x802C, 0x384B54463531323634485A2D314736453120
AirPort: spairport_wireless_card_type_airport_extreme (0x14E4, 0x112), Broadcom BCM43xx 1.0 (7.15.166.24.3)
Bluetooth: Version 4.3.5f8 15969, 3 services, 27 devices, 1 incoming serial ports
Network Service: Wi-Fi, AirPort, en0
PCI Card: Apple 57761-B0, Ethernet Controller, Thunderbolt@191,0,0
PCI Card: pci11c1,5901, IEEE 1394 Open HCI, Thunderbolt@190,0,0
PCI Card: pci12d8,400e, USB Open Host Controller, Thunderbolt@196,0,0
PCI Card: pci12d8,400e, USB Open Host Controller, Thunderbolt@196,0,1
PCI Card: pci12d8,400f, USB Enhanced Host Controller, Thunderbolt@196,0,2
Serial ATA Device: APPLE SSD SD0256F, 251 GB
USB Device: Hub
USB Device: MFC-250C
USB Device: FaceTime HD Camera (Display)
USB Device: Display Audio
USB Device: Apple Thunderbolt Display
USB Device: Internal Memory Card Reader
USB Device: BRCM20702 Hub
USB Device: Bluetooth USB Host Controller
USB Device: Apple Internal Keyboard / Trackpad
Thunderbolt Bus: MacBook Pro, Apple Inc., 17.2
Thunderbolt Device: Thunderbolt Display, Apple Inc., 1, 26.2

**Comment by bawigga** [comment]
+1

**Comment by joelind** [comment]
+1

**Comment by wernerlaude** [comment]
http://www.psequel.com/download?version=1.1.2
works..

**Comment by psequel** [comment]
Fixed in v1.2.1.beta. See https://twitter.com/psequel/status/619029292488376320
